drush-services.yml

```diff
@@ -0,0 +1,42 @@
+#
+# Drush services
```

Comment:
I see that this services file is hard-coded Drush’s root dir. If we support any user editing of it, we need to relocate it since:

1. .phar users can’t edit inside the phar
1. Sometimes there 2 Drush’s that service a request. Users will get confused
1. composer install/update will wipe out a Drush in a vendor dir